src/System.Globalization/tests/Performance/Perf.CompareInfo.cs

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using Microsoft.Xunit.Performance;
+
+namespace System.Globalization.Tests
+{
+    public class Perf_CompareInfo
+    {
+        [Benchmark]
+        [InlineData("string1", "string2", CompareOptions.None)]
+        [InlineData("StrIng", "string", CompareOptions.IgnoreCase)]
+        [InlineData("StrIng", "string", CompareOptions.OrdinalIgnoreCase)]
+        [InlineData("\u3060", "\u305F", CompareOptions.None)]
+        [InlineData("ABCDE", "c", CompareOptions.None)]
+        [InlineData("$", "&", CompareOptions.IgnoreSymbols)]
+        public void Compare(string string1, string string2, CompareOptions options)
+        {
+            CompareInfo compareInfo = CultureInfo.CurrentCulture.CompareInfo;
```

Comment:
> CultureInfo.CurrentCulture.CompareInfo; [](start = 37, length = 40)

using CurrentCulture would be fragile. I suggest in these tests we pass the culture info objects to the test methods. we can use en-US, Spanish, Turkish and Japanese cultures.